app/src/test/java/ar/scacchipa/twittercloneapp/domain/AuthorizationUseCaseTest.kt

```diff
@@ -48,7 +48,11 @@ class MockAuthorizationRespository: IAuthorizationRepository {
         )
     }
 
-    override fun getErrorUserCaseTokenCreator(): UserAccessToken {
-        return UserAccessToken(error = "error")
+    override fun getCancelledAuthToken(): UserAccessToken {
+        return UserAccessToken(error = "cancelled_authorization")
+    }
+
+    override fun getErrorLookupToken(): UserAccessToken {
```

Comment:
same as before, I think it should not reach the repository this functionality.

Reply:
I was removed and the object is created in the class.